### ISSUE TITLE
Scroll event

### ISSUE THREAD

**Issue by fiznux** [HsuanXyz/ion2-calendar#14]
Can we subcribe to scroll event ? upward and backward ?

**Comment by HsuanXyz** [comment]
async daysConfig？

**Comment by fiznux** [comment]
yes correct ! it would be great if you could implemented that

**Comment by HsuanXyz** [comment]
>English is not my native language; please excuse typing errors.

Good idea!  I am refactoring the style, and then I will consider joining this feature.

**Comment by serra82** [comment]
interesting feature, any news?!?
